example/fittemplate.md

```diff
@@ -0,0 +1,137 @@
+
+In this example we will show two ways on how to create an individual head model on the basis of surface
+data acquired with the Polhemus. Both methods determine the translation, rotation and scaling of the 
+template head model to fit the Polhemus head shape, but the second method applies a more detailed spatial scaling.
+
+## Download
+
+Both approaches will be demonstrated using the same dataset, which you can find on the 
+[ftp server](ftp://ftp.fieldtriptoolbox.org/pub/fieldtrip/tutorial/epilepsy). More 
+information on this dataset can be found [here](/tutorial/epilepsy/).
+
+## Loading the data
+
+Before starting with FieldTrip, it is important that you set up your
+[MATLAB path](/faq/should_i_add_fieldtrip_with_all_subdirectories_to_my_matlab_path) properly.
+
+You can load the head shape measured during the MEG recording with the Polhemus and a
+template volume conduction model. We have to ensure that they have consistent units, hence 
+we will convert the units into mm.
+
+    polhemus = ft_read_headshape(filename);
+    polhemus = ft_convert_units(polhemus,'mm');
+
+    template = ft_read_vol('standard_bem.mat');
+    template = ft_convert_units(template,'mm');
+
+## Coregistration
+
+In the next step we coregister the template head model with the Polhemus head shape to ensure that they are expressed consistently in the MEG coordinate system (i.e. relative to the same origin and with the axes pointing in the same direction). The coregistration consists of a rotation and translation.
+
+    cfg = [];
+    cfg.template.headshape      = polhemus;
+    cfg.checksize               = inf;
+    cfg.individual.headmodel    = template;
+    cfg                         = ft_interactiverealign(cfg);
+    template                    = ft_transform_geometry(cfg.m, template);
+
+## Determining and applying of the transformation
+
+### Method 1: On the basis of fiducials
+
+For the first approach we will fit a sphere to the three fiducials and we will fit another sphere to the template head model. On the basis of difference of the center and scaling between the two spheres we can derive a transformation containing information of the translation and the global scaling.
+
+For this approach we will need determine the fiducials to Fiducials
+
+    nas = polhemus.elecpos
+    lpa = polhemus.elecpos
+    rpa = polhemus.elecpos
+
+Fit a sphere to the MRI template
+
+    cfg=[];
+    cfg.method='singlesphere';
+    scalp_sphere = ft_prepare_headmodel(cfg, template.bnd(1));
+
+Fit a sphere to the fiducials
+
+    cfg=[];
+    cfg.method = 'singlesphere';
+    headshape_sphere = ft_prepare_headmodel(cfg, fiducials);
```

Comment:
I don't think this actually works. Did you test the code? Fitting a sphere to three points is ill-defined, it does not constrain the radius.

Reply:
I misunderstood what the script from Weiyong was doing. I now use the Polhemus measurement (after applying ft_defacemesh)